code/modules/client/preference_setup/preferences_sql.dm

```diff
@@ -162,16 +161,21 @@
 			for (var/variable in var_names)
 				if (isnull(var_names[variable]))
 					query += " [variable] = [arg_names[i]]"
-				if (i < var_names.len)
-					query += ", "
+
+					if (i < var_names.len)
+						query += ","
 
 				i++
 
+			query = replacetext(query, ",", "", length(query) - 1)
```

Comment:
No really better way to do this, tbh. At least it only checks 1 character at a given index, so it's not that costly.